docs/references/express/migrate-from-node-to-express.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Clerk Express SDK
+description: Learn how to migrate from Clerk Node SDK to Express SDK.
+---
+
+# Clerk Express SDK
+
+## Migrating from Clerk Node SDK to Clerk Express SDK
+
+<Steps>
+
+### Uninstall @clerk/clerk-sdk-node and install @clerk/express
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm uninstall @clerk/clerk-sdk-node
+```
+
+```sh filename="terminal"
+yarn remove @clerk/clerk-sdk-node
+```
+
+```sh filename="terminal"
+pnpm remove @clerk/clerk-sdk-node
+```
+
+</CodeBlockTabs>
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm install @clerk/express
+```
+
+```sh filename="terminal"
+yarn add @clerk/express
+```
+
+```sh filename="terminal"
+pnpm add @clerk/express
+```
+</CodeBlockTabs>
+### Require Authentication
+
+**Using Node SDK**
+
+To enforce strict authentication, use `ClerkExpressRequireAuth()`. This middleware will raise an error when an unauthenticated request is made.
+You can find an example [here](/docs/backend-requests/handling/nodejs#clerk-express-require-auth).
+
+**Using Express SDK**
+
+- **Centralized Middleware:**
+Use `clerkMiddleware` to authenticate all incoming requests without blocking them. This middleware will handle the handshake mechanism.
+- **Require Authentication in Specific Routes or Handlers:**
+Use `requireAuth` to enforce authentication on specific routes. This middleware returns an HTTP 401 response for unauthenticated request.
+```js
+const express = require('express');
+const { clerkMiddleware, requireAuth } = require('@clerk/express');
+
+const app = express();
+
+// Apply centralized middleware
+app.use(clerkMiddleware());
+
+// Define a protected route
+app.get('/protected', requireAuth, (req, res) => {
+  res.send('This is a protected route');
+});
+
+app.listen(3000, () => {
+  console.log('Server is running on port 3000');
+});
+```
```

Comment:
We need to provide an example for how to exactly replicate the behavior that was the previous default. This is the starting point with migrations that we try to present for users - first you figure out how can you upgrade the package while retaining the same behavior to make sure it works, then you think about whether you'd like to change the behavior based on the new package's capabilities.

Reply:
The examples in the current docs are replicating the exact same behaviour with the one of `@clerk/clerk-sdk-node`. Could you elaborate more on your comment?